01-node-tutorial/answers/12-http.js

```diff
@@ -0,0 +1,19 @@
+const http = require('http');
+
+const server = http.createServer((req, res) => {
+    if (req.url === '/') {
+        res.end('Welcome to our home page');
```

Comment:
don't forget to also write the response:

```suggestion
        res.write('Welcome to our home page');
        res.end();
```